pymtl3/passes/backends/verilog/translation/behavioral/VBehavioralTranslatorL2.py

```diff
@@ -130,11 +130,12 @@ def visit_For( s, node ):
 
     begin    = ' begin' if len( node.body ) > 1 else ''
 
-    cmp_op   = '<' if node.step.value > 0 else '<'
-    inc_op   = '+' if node.step.value > 0 else '-'
+    cmp_op   = '>' if node.step._value[31] else '<'
```

Comment:
how does this _value[31] work? inferred as Bits32?

Reply:
Yes -- checking if the number is negative by looking at the MSB